### ISSUE TITLE
Meteor 1.7 error in server log

### ISSUE THREAD

**Issue by schlaegerz** [abecks/meteor-inject-data#6]
Seems for some reason the headers varibale is undefined in Meteor 1.7

error on fast-rendering path: /reset-password ; error: TypeError: Cannot read property '_injectPayload' of undefined
W20180530-09:48:12.562(-7)? (STDERR)     at Object.getData (packages/staringatlights:inject-data/lib/server.js:63:18)
W20180530-09:48:12.562(-7)? (STDERR)     at packages/staringatlights_fast-render.js:212:38
W20180530-09:48:12.563(-7)? (STDERR)     at packages/staringatlights_fast-render.js:288:7

**Comment by TNick** [comment]
I got this error while implementing tests.
```bash
export TEST_BROWSER_DRIVER=phantomjs 
export TEST_WATCH=1 
meteor test --driver-package meteortesting:mocha --port 3051
```

 What I did was:
- create a `packages` directory in my app, 
- downloaded the source from this repository and placed it in `packages/staringatlights_inject-data`
- modified [this line](https://github.com/abecks/meteor-inject-data/blob/master/lib/server.js#L63) so that it checks for the presence of headers:

```javascript
        // server.js line 63
	if (req.headers && req.headers._injectPayload) {
```
- also had to get rid of the entire `onTest` in `package,js` but that might be because of my custom setup.

**Comment by alexismoreau** [comment]
Thanks @TNick, that fixed it.

For the error:

> package.js:30:6: Duplicate source file: tests/init.js

you should just remove the line : 
```
// package.js line 27 
api.addFiles(['tests/init.js'], 'server')
```
in package.js

I didn't check if the tests run.

Can you open a pull request with these changes ?

**Comment by para1igal** [comment]
+1 I'm still seeing this error on the project. Any progress with getting it fixed?

**Comment by abecks** [comment]
Should be fixed in 2.1.1 now. Thanks everyone.
